libs/features/georoutingv2/georoutingv2.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/no-cycle */
+import { sendAnalytics } from '../../blocks/modal/modal.js';
```

Comment:
I know this is to keep the code dry, but I do not think georoutingv2 should have a dependency on modal.js / block code.  Could we move the send analytics function somewhere else that bot georoutingv2 and modal.js could consume?

Reply:
@JasonHowellSlavin thanks for your review. However modal.js is already being imported in georouting file so it already has dependency on modal block code which I feel is needed and cannot be removed.

Reply:
Right you are! 

If we're importing at the top, does it make sense then to import `getModal` at the same time, and refactor the `showModal` function?

Reply:
Hi @JasonHowellSlavin I went ahead and did the other way round. Removed import from top of the file and added sendAnalytics function to the existing import call. Please let me know if that looks good